packages/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -522,6 +524,30 @@ class Transforms {
           }
         }
 
+        // Decrypt encrypted text map if present
+        if (activity.encryptedTextKeyValues !== undefined) {
+          for (const [key, value] of Object.entries(activity.encryptedTextKeyValues)) {
```

Comment:
suggestion: minor point, but the `forEach` construction when iterating Object.entries is more common in this codebase. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries#iterating_through_an_object